README.md

```diff
@@ -6,9 +6,66 @@
 
 - [Shortcuts for @operatehappy](#shortcuts-for-operatehappy)
   - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Dependencies](#dependencies)
+  - [Usage](#usage)
   - [Author Information](#author-information)
   - [License](#license)
 
+## Requirements
+
+This module requires Node.js 6.x or later.
```

Comment:
glad to see you support people who hate themselves.

Reply:
Technically, I'm only using `npm` and `node` as a delivery mechanism; This could be solved using `brew` or other things, but this is beautifully simple ;-)